Scalar nan

felipeaoli opened the pull request:
Fixing the incompatibility between ast function and ".nan" found in yaml.

felipelm commented:
@felipeaoli can you test it again after @crisingulani changes?

Maybe this is not necessary anymore.

felipeaoli commented:
I removed the ast function and merged with the master but the it seems to not have completed the transition to the dB reading. Then I reverted those of my last changes in order to keep it working.
